ts/BUILD.typescript

```diff
@@ -0,0 +1,42 @@
+load("@aspect_bazel_lib//lib:copy_directory.bzl", "copy_directory")
+load("@aspect_rules_js//js:nodejs_package.bzl", "nodejs_package")
+load("@aspect_rules_js//js:nodejs_binary.bzl", "nodejs_binary")
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+
+copy_directory(
+    name = "_npm_typescript",
+    src = "package",
+    out = "typescript",
+)
+
+nodejs_package(
+    name = "npm_typescript",
+    src = "_npm_typescript",
+    package_name = "typescript",
+    visibility = ["//visibility:public"],
+    deps = [],
```

Comment:
Is this required? 🤷

Reply:
no, the empty deps line is not meaningful, just came from copying this out of a generated file, will remove